src/main/java/seedu/address/storage/JsonAdaptedClient.java

```diff
@@ -33,28 +32,25 @@
 
     private final String name;
     private final String phone;
-
     private final String email;
     private final String address;
     private final String country;
     private final String timezone;
     private final String contractExpiryDate;
     private final String lastModifiedInstant;
-    private final List<JsonAdaptedTag> tagged = new ArrayList<>();
     private final List<JsonAdaptedNote> clientNotes = new ArrayList<>();
 
     /**
      * Constructs a {@code JsonAdaptedClient} with the given client details.
      */
     @JsonCreator
     public JsonAdaptedClient(@JsonProperty("name") String name,
-                @JsonProperty("phone") String phone, @JsonProperty("email") String email,
-                @JsonProperty("address") String address, @JsonProperty("country") String country,
-                @JsonProperty("timezone") String timezone,
-                @JsonProperty("contractExpiryDate") String contractExpiryDate,
-                @JsonProperty("lastModifiedInstant") String lastModifiedInstant,
-                @JsonProperty("tagged") List<JsonAdaptedTag> tagged,
-                @JsonProperty("clientNotes") List<JsonAdaptedNote> clientNotes) {
+                             @JsonProperty("phone") String phone, @JsonProperty("email") String email,
+                             @JsonProperty("address") String address, @JsonProperty("country") String country,
+                             @JsonProperty("timezone") String timezone,
+                             @JsonProperty("contractExpiryDate") String contractExpiryDate,
+                             @JsonProperty("lastModifiedInstant") String lastModifiedInstant,
+                             @JsonProperty("clientNotes") List<JsonAdaptedNote> clientNotes) {
```

Comment:
change back the indentation pls, supposed to be 8 spaces after the primary line, not 16

Reply:
done